lib/init.php

```diff
@@ -30,7 +30,9 @@ function gutenberg_menu() {
 		'gutenberg'
 	);
 
-	if ( get_option( 'gutenberg-experiments' ) ) {
+	// The navigation editor menu item is temporarily removed by hardcoding `false` here.
+	// See https://github.com/WordPress/gutenberg/pull/40878 for more info.
+	if ( false && get_option( 'gutenberg-experiments' ) ) {
```

Comment:
This introduces a dead code branch. How about removing lines 35-46 entirely?

Reply:
I agree with @adamziel that we should remove this code as well since we removed the option to opt in, the menu item code is of no use, had we surrounded with a dead branch condition the `add_settings_field ` code then we could argue we may come back to it, but given that is removed this other bit should also be just removed.

Reply:
That's just it though. We don't necessarily want to remove this code. All we're doing here is hiding the experiment until such time as we (may/may not) decide to remove the Navigation screen project.

I'm not convinced we're ready to do that. Neither is @talldan. That's why this was aiming to be a simple "hide the experiment" PR 😄

Reply:
I agree that we can remove this _specific_ piece of code however. We don't need to have it sitting around and it can be rolled back.

What I'm saying is that the decision has not been made to remove the entire Navigation Editor so I won't be removing that in this (or any other) PR...yet.

Reply:
> What I'm saying is that the decision has not been made to remove the entire Navigation Editor so I won't be removing that in this (or any other) PR...yet.

Sure! Let's retain the feature and let's let these ten lines go :-)

Reply:
Ok I've removed the dead code branch.